cvat-ui/src/components/actions-menu/dump-submenu.tsx

```diff
@@ -30,17 +30,21 @@ export default function DumpSubmenu(props: Props): JSX.Element {
     return (
         <Menu.SubMenu key={menuKey} title='Dump annotations'>
             {
-                dumpers.map((dumper: string): JSX.Element => {
-                    const pending = (dumpActivities || []).includes(dumper);
-                    const isDefault = isDefaultFormat(dumper, taskMode);
+                dumpers
+                    .sort((a: any, b: any) => a.name.localeCompare(b.name))
+                    .map((dumper: any): JSX.Element =>
+                {
+                    const pending = (dumpActivities || []).includes(dumper.name);
+                    const disabled = !dumper.enabled || pending;
+                    const isDefault = isDefaultFormat(dumper.name, taskMode);
                     return (
                         <Menu.Item
-                            key={dumper}
-                            disabled={pending}
+                            key={dumper.name}
+                            disabled={disabled}
                             className='cvat-menu-dump-submenu-item'
                         >
                             <Icon type='download' />
-                            <Text strong={isDefault}>{dumper}</Text>
+                            <Text strong={isDefault} disabled={disabled}>{dumper.name}</Text>
```

Comment:
You can try something like that: ``type={disabled ? 'secondary' : undefined}`` if you want to do text gray. 
But I would suggest do not display such formats. Just using:
```
dumpers.filter((dumper: any) => dumper.enabled).sort(...
```

Reply:
One more option is to use style attribute:
```
<Text style={disabled ? { opacity: '0.5' } : {}} ...
```

Reply:
I tried to just pass `disabled` to  `Icon` element, but it didn't work, while it is described in the documentation of the library. I guess, we can be using an outdated version.

Reply:
@zhiltsov-max 

We use version 3.x. It was the latest version when we started doing this UI. 
Now version 4.x exists, but migrating is painful